package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Legal Aid",
   "main": "index.js",
   "scripts": {
-    "lint": "gulp eslint",
+    "lint": "node node_modules/gulp/bin/gulp eslint",
```

Comment:
Yes. My check is why you're using `node node_modules/gulp/bin/` in the first place. We want it to be simply.